tests/unit-tests/ambiguous/__snapshots__/arrow-parens-avoid.test.ts.snap

```diff
@@ -68,7 +68,7 @@ const bool = false,
 exports[`ambiguous > config > arrowParens: "avoid" > (oh, no) => {} > with semi, with newline > it formats ../cases/gjs/simple.gjs 1`] = `
 "<template>
   what
-</template>;
+</template>
 (oh, no) => {};
```

Comment:
this will not cause syntax errors anymore, as content-tag uses a real ast for transforms instead of just string replacements

Reply:
wrong, it will cause syntax errors...